plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -2282,14 +2242,22 @@ void read_write::send_transaction2(const read_write::send_transaction2_params& p
                            }
                         } );
                   } else {
-                     fc::variant output;
-                     try {
-                        output = db.to_variant_with_abi( *trx_trace_ptr, abi_serializer::create_yield_function( abi_serializer_max_time ) );
-                     } catch( chain::abi_exception& ) {
-                        output = *trx_trace_ptr;
-                     }
-                     const chain::transaction_id_type& id = trx_trace_ptr->id;
-                     next( read_write::send_transaction_results{id, std::move( output )} );
+                     auto yield        = abi_serializer::create_yield_function(fc::microseconds::maximum());
```

Comment:
Add a comment here indicating it is done off main thread.

Reply:
Done, thank you for the suggestion.